implementations/rust/ockam/ockam_command/src/run/demo_config_files/4.kafka.inlet.yaml

```diff
@@ -0,0 +1,6 @@
+name: kafka_inlet_node
```

Comment:
In the `run` command nodes are described as regular resources:

```yaml
nodes: kafka_inlet_node
```

The `parse_demo_config_files` test won't fail because it will skip the `name` field, but if you look into what it got parsed, you'll see no nodes.